### ISSUE TITLE
E: Unable to locate package linux-image-armmp

### ISSUE THREAD

**Issue by anthhov** [TheSin-/rpi-img-builder#34]
Trying to build Debian image based on _armel_ for RPi 2:
`sudo make REPO=Debian DIST_ARCH=armel ARCH=armmp`
At some moment _make_ crashes saying:
`E: Unable to locate package linux-image-armmp`

After looking at [Debian packages](https://packages.debian.org/stretch/linux-image-armmp) page I mentioned that there is really no _linux-image-armmp_ package for _armel_ architecture, only for _armhf_. What I did:
1. Added `multiarch=armhf armel` line to _multistrap.list.in_
2. Edited `linux-image-__ARCH__` line in _plugins/Debian/kernel/packages_ to `linux-image-__ARCH__:armhf`

After that I successfully built the image and flashed it via Etcher to MicroSD card. Tomorrow I will check if this image works on RPi2. In this case I can modify the script to work with _armel_ based images and get linux kernel from packages for _armhf_.

**Comment by TheSin-** [comment]
drop ARCH=armmp

**Comment by TheSin-** [comment]
armel != armhf do not mix those bad things will happen.

**Comment by aostrouhhov** [comment]
> drop ARCH=armmp

If I do, it will look for _linux-image-armel_, which also does not exist in Debian packages.

> armel != armhf

In fact, I have one RPi 2 which has _armel_ Debian distribution with soft-float ABI on it. But _uname -a_ has the following words: `Linux 4.1.7-v7+ armv7l GNU/Linux`. As far as I understand, this is _armel_ Debian distribution with _armhf_ linux kernel.

**Comment by TheSin-** [comment]
that is because I believe it was removed with jessie, if you want true armel you'll never to set het dist to jessie IIRC.  It's been a while I have an original CM that I used to have to build armel for, but I've updated to CM3 now so I haven't done it since stretch went stable.

either way feel free to do what you are doing but I won't be adding it as it's too risky, I could add it so when armel is used that it forces jessie though.
